src/rules.mk

```
@@ -241,6 +231,6 @@ $(QA_DIR)/%.git-log.csv: %.rst
 	@ $(GIT_LOG) '$<' '$@'
 
 .PHONY: qa
-qa: qa-deps $(git_log_targets)
+qa: qa-deps vale
```

Don't know for sure. Should we use `lint-report` here?

```suggestion
qa: qa-deps lint-report
```

---

this will mean that running `make qa` will also run `make lint-report`. is that the intended behaviour?

---

Not sure. You are the architect ;].

---

> Maybe just an unqualified suggestion from my end.

Effectively, `make qa` would only run `make check`, right? That seemed redundant to me and I believed the QA stuff would be about the telemetry thing, which is now covered by `lint-report`.